src/index.js

```diff
@@ -45,12 +47,12 @@ class Client {
       throw new Error(`Unexpected response from LFS server: ${response.status}`);
     }
 
-    return response;
+    return response.data;
```

Comment:
we will need to return `response` it self, cause in the upload implementation we check the response status code

Reply:
There will need only the status code besides the data? 
Because return the entire response from this HTTP method isn't the best approach... 
If will be needed just the data and the status I can destruct the response and send just that.

Reply:
Yes there is a need for the status. 

it will be good if you destruct it

Reply:
ok, I will update to return just the data and the status, ok?